app/services/container_dashboard_service.rb

```diff
@@ -123,6 +123,7 @@ def heatmaps
     # Get latest hourly rollup for each node.
     node_ids = @ems.container_nodes if @ems.present?
     metrics = MetricRollup.latest_rollups(ContainerNode.name, node_ids)
+    metrics = metrics.where('timestamp > ?', 1.day.ago.utc).includes(:resource)
```

Comment:
```DISTINCT``` in ```latest_rollups``` should stay there since you dont want 2 rollups for the same resource.
I think that ```includes(:resource)``` should be here since whoever uses ```latest_rollups``` might not always need it.

I use the ```includes(:resource)``` because I need ```metric.resource.name``` for each metric. I remember attempting to select that part only in #5938 but being unsuccessful at it.